classes/create.rb

```diff
@@ -0,0 +1,62 @@
+require './classes/student'
+require './classes/teacher'
+require './classes/book'
+require './classes/rental'
+module Create
+  def check_response(response)
+    false if %w[n N].include? response
+    true if %w[y Y].include? response
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = check_response(gets.chomp)
+    @persons.push(Student.new(@classroom, age, name, parent_permission: parent_permission))
+    puts "Student has been successfully created! \n\n"
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @persons.push(Teacher.new(specialization, age, name, parent_permission: true))
+    puts "Teacher has been sucessfully created! \n\n"
```

Comment:
- [ ] I think it will be better if all function follows the SOLID principle, especially the single responsibility principle meaning that each method should have one responsibility. In your code, I can see that this particular method is receiving user input and also creating a teacher. Kindly create a separate function for the different responsibilities 😄

Reply:
Noted thank you